tts/scripts/tts_node.py

```diff
@@ -102,8 +105,16 @@ def do_speak(goal):
 
     if 'Audio File' in r:
         audio_file = r['Audio File']
+        # Soundplay cannot handle more than 32 different file paths.
+        # Creating a symlink to the polly output and then passing that to
+        # to the soundplay node is a workaround.
+        # See: https://github.com/ros-drivers/audio_common/issues/125
+        sym_link_path = os.path.abspath(SOUND_FILE_PATH)
+        os.remove(sym_link_path)
```

Comment:
Wouldn't this throw an exception the first time? (before the link exists)

Reply:
Turns out it does. Fixed